modules/theme/src/themes/dark/assets/images/icons/plus-icon.svg

```diff
@@ -1,5 +1,5 @@
 <!--
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Should be `2019` since the intial file is added in 2019

Reply:
The previous icon replaced with a new icon.

Reply:
Fixed